ProductionModePlugin.js

```diff
@@ -67,34 +67,37 @@ class ProductionModePlugin {
         if(param.isString() && param.string === "globalize" && this.moduleFilter(request) &&
           !(globalizeCompilerHelper.isCompiledDataModule(request))) {
 
-          // Statically extract Globalize formatters and parsers from the request
-          // file only. Then, create a custom precompiled formatters/parsers module
-          // that will be called instead of Globalize, which in turn requires
-          // Globalize, set the default locale and then exports the Globalize
-          // object.
-          const compiledDataFilepath = globalizeCompilerHelper.createCompiledDataModule(request);
+          this.supportedLocales.forEach((locale) => {
```

Comment:
> We now create a set of compiled data modules per locale, enabling webpack to allocate them distinct Ids.

Today, GWP creates one set of files (let's say moduleId 100 to 105 if you have 6 files with `const Globalize = …`). Later on, it [reuses](https://github.com/rxaviers/globalize-webpack-plugin/blob/master/ProductionModePlugin.js#L261) these Ids for all modules. In the end, all the i18n chunks share the same moduleIds.
If you try to load more than one of them, only the first one is loaded by webpack as its loader cache mecanism thinks the modules are already loaded.
In order to have unique moduleIds, GWP now creates one file for all the combinations (**but** [only the moduleId](https://github.com/rxaviers/globalize-webpack-plugin/pull/65/commits/a8ac37672ab9db4ddf593cc84d61234a46c64193#diff-33e35b45cee16aa7f9fc8f47ab0e2419R96) of the matching development locale module replaces `"globalize"`.) I wish I knew how to create "virtual" modules (non-file backed)…